views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,18 @@
+
+<div class="desc-img">
+
+    <img src="{{image_url}}" alt="beer bottle">
+
+</div>
+
+<h2>
+    {{name}}
+</h2>
+
+<p>
+    {{description}}
+</p>
+
+<p>
+    {{tagline}}
+</p>
```

Comment:
A small thing, but it would be nice to have the info introduced, as opposed to just pasted in there. So for instance:

Brewer's tips: {{ brewers_tips}}